packages/angular-workspace/nimble-angular/table-column/date-text/nimble-table-column-date-text.directive.ts

```diff
@@ -22,22 +22,22 @@ import {
 } from '@ni/nimble-components/dist/esm/table-column/date-text/types';
 
 export type { TableColumnDateText };
-export { DateStyle };
-export { DateTextFormat };
-export { DayFormat };
-export { DayPeriodFormat };
-export { EraFormat };
-export { FormatMatcherAlgorithm };
-export { HourCycleFormat };
-export { HourFormat };
-export { LocaleMatcherAlgorithm };
-export { MinuteFormat };
-export { MonthFormat };
-export { SecondFormat };
-export { TimeStyle };
-export { TimeZoneNameFormat };
-export { WeekdayFormat };
-export { YearFormat };
+export type { DateStyle };
+export type { DateTextFormat };
+export type { DayFormat };
+export type { DayPeriodFormat };
+export type { EraFormat };
+export type { FormatMatcherAlgorithm };
+export type { HourCycleFormat };
+export type { HourFormat };
+export type { LocaleMatcherAlgorithm };
+export type { MinuteFormat };
+export type { MonthFormat };
+export type { SecondFormat };
+export type { TimeStyle };
+export type { TimeZoneNameFormat };
+export type { WeekdayFormat };
+export type { YearFormat };
```

Comment:
This is reverting an explicit change from before, seems fishy: https://github.com/ni/nimble/pull/1438

Reply:
We should make sure to build tgzs against SystemLinkShared and an app to reduce the chance we are regressing against something we expect to be exporting the value of and not just type.

Reply:
I agree that some more testing would be good. It looks like there are a number of places where something that had been exported now only has its type exported. Each of those instances is a possible place where a client could be broken depending on how they are using the exported entity.

Reply:
Ran a SystemLinkShared pipeline build against these changes, and everything looks good: https://ni.visualstudio.com/DevCentral/_build/results?buildId=12189861

Reply:
There is really only one interesting change other than the tsconfig files. I've added a comment to each of those.